packages/ckeditor5-dev-web-crawler/src/getlinks.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* eslint-env browser */
+// eslint-disable-next-line spaced-comment
+/// <reference lib="dom" />
```

Comment:
TBD: To add `DOM` as a known library in tsconfig.

Reply:
Self answer: It would allow the use of DOM API in a Node.js package. While it does not make sense for every file except this one, as `getlinks` is executed in the browser context.